linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -1,12 +1,11 @@
 package io.buoyant.linkerd.admin
 
-import com.twitter.finagle.http.{MediaType, Request, Response}
-import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.http.{MediaType, Response}
 import com.twitter.util.Future
 import io.buoyant.admin.HtmlView
 import io.buoyant.linkerd.Build
 
-object AdminHandler extends HtmlView {
+class AdminHandler(namerdNav: Boolean) extends HtmlView {
```

Comment:
This may be moot given alex's ticket above, but consider just passing `namerdNav` into the `html` method, and generating the navBar using that. This way, `AdminHandler` can go back to being an object. Or, consider moving all the methods that do not depend on state into an object.